frontend/www/js/omegaup/problem/collection_list_by_author.ts

```diff
@@ -0,0 +1,82 @@
+import Vue from 'vue';
+import problem_CollectionListAuthor from '../components/problem/CollectionListAuthor.vue';
+import { types } from '../api_types';
+import { omegaup, OmegaUp } from '../omegaup';
+import * as ui from '../ui';
+
+OmegaUp.on('ready', () => {
+  const payload = types.payloadParsers.CollectionDetailsByAuthorPayload();
+  const queryString = window.location.search;
+  let sortOrder = 'desc';
+  let columnName = 'problem_id';
+  let language = 'all';
+  let query = '';
+  if (queryString) {
+    const urlParams = new URLSearchParams(queryString);
+    if (urlParams.get('sort_order')) {
+      const sortOrderParam = urlParams.get('sort_order');
+      if (sortOrderParam) {
+        sortOrder = sortOrderParam;
+      }
+    }
+    if (urlParams.get('order_by')) {
+      const columnNameParam = urlParams.get('order_by');
+      if (columnNameParam) {
+        columnName = columnNameParam;
+      }
+    }
+    if (urlParams.get('language')) {
+      const languageParam = urlParams.get('language');
+      if (languageParam) {
+        language = languageParam;
+      }
+    }
+    if (urlParams.get('query')) {
+      const queryParam = urlParams.get('query');
+      if (queryParam) {
+        query = queryParam;
+      }
+    }
+  }
+  new Vue({
+    el: '#main-container',
+    components: {
+      'omegaup-problem-collection-list-author': problem_CollectionListAuthor,
+    },
+    render: function (createElement) {
+      return createElement('omegaup-problem-collection-list-author', {
+        props: {
+          data: payload,
+          problems: payload.problems,
+          loggedIn: payload.loggedIn,
+          currentTags: payload.currentTags,
+          pagerItems: payload.pagerItems,
+          wizardTags: payload.tagData,
+          language: payload.language,
+          languages: payload.languages,
+          keyword: payload.keyword,
+          sortOrder: sortOrder,
+          columnName: columnName,
+        },
+        on: {
+          'apply-filter': (
+            columnName: string,
+            sortOrder: omegaup.SortOrder,
+          ): void => {
+            const queryParameters = {
+              language,
+              query,
+              order_by: columnName,
+              sort_order: sortOrder,
+            };
+            window.location.replace(
+              `/problem/collection/author/?${ui.buildURLQuery(
+                queryParameters,
+              )}`,
```

Comment:
```suggestion
              )}/`,
```

hay que intentar siempre ponerle el `/` al final de todos los URLs.